raiden-dapp/tests/unit/raiden-service.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('RaidenService', () => {
     expect.assertions(1);
     providerMock.mockResolvedValue(mockProvider);
     factory.mockResolvedValue(mockRaiden());
-    await raidenService.connect();
+    await raidenService.connect('');
```

Comment:
From looking at `raidenService.connect` it looks like the `stateBackup` is optional
I think it would be nicer to not have to pass an empty string to it. If really necessary one could use [default parameters](https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters) as an alternative